client/me/purchases/purchases-list-in-dataviews/index.tsx

```diff
@@ -48,13 +52,14 @@ export interface PurchasesListProps {
 export interface PurchasesListConnectedProps {
 	hasLoadedUserPurchasesFromServer: boolean;
 	isFetchingUserPurchases: boolean;
-	purchases: Purchase[] | null;
+	purchases: Purchase[];
 	subscriptions: MembershipSubscription[];
 	sites: SiteDetails[];
 	nextAppointment: NextAppointment | null;
 	isUserBlocked: boolean;
 	availableSessions: number[];
 	siteId: number | null;
+	userId: number | undefined;
```

Comment:
Why undefined instead of null?

Also, if undefined, then we can write `userId?: number` I suppose